### PR TITLE
Ensure empty box is still an empty box after a Box3.applyTransform

### LINKED ISSUE #8620: Default Box3 multiplied by Matrix4 is not empty()
##### Description of the problem 

If all of the coordinates in `Box3` are set to `NaN`, then `empty()` returns false. This is troublesome for cases where we use `applyMatrix4()` transformations to empty boxes and expect them to remain empty.

##### Three.js version

- [ ] Dev
- [x] r75
- [x] r73

##### Browser

- [ ] All of them
- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [x] Windows
- [ ] Linux
- [ ] Android
- [ ] IOS

### PR DESCRIPTION
**Pull request by bhouston**
Fixes #8620.

### CODE REVIEW

File: src/math/Box3.js [hunk 1]
```diff
@@ -423,7 +426,6 @@ THREE.Box3.prototype = {
 			points[ 6 ].set( this.max.x, this.max.y, this.min.z ).applyMatrix4( matrix ); // 110
 			points[ 7 ].set( this.max.x, this.max.y, this.max.z ).applyMatrix4( matrix );	// 111
 
-			this.makeEmpty();
```
Comment (tschw) [inline src/math/Box3.js]: Note: This is redundant and called again in `setFromPoints`.

**Comment by tschw** [comment]
My fix (documenting a limitation) already got merged, so you probably should remove that line from the docs in this one.

**Comment by bhouston** [comment]
Thanks @tschw got the docs addressed.

**Comment by tschw** [comment]
Looks ready, then.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**